release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,82 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, registry overrides configured by a cluster administrator on the management side applied to non-relevant data-plane components. With this release, registry overrides no longer apply to these components.(link:https://issues.redhat.com/browse/OCPBUGS-35549[*OCPBUGS-35549*]).
+
+* Previously, the OpenShift Cluster Manager container did not have the right TLS Certificates. As a result, image streams could not be used in disconnected deployments. With this update, the TLS Certificates are added as projected volumes. (link:https://issues.redhat.com/browse/OCPBUGS-35482[*OCPBUGS-35482*])
+
+* Previously, in disconnected environments, the HyperShift Operator ignored registry overrides. As a consequence, changes to node pools were ignored, and node pools encountered errors. With this update, the metadata inspector works as expected during the HyperShift Operator reconciliation, and the override images are properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-35401[*OCPBUGS-35401*])
+
+* Previously, the secrets-store CSI driver on Hypershift was failing to mount secrets due to an issue with the Hypershift CLI. With this release, the driver is able to mount volumes and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35183[*OCPBUGS-35183*])
+
+* Previously, the reduction of the network queue did not work as expected for inverted rules such as `!ens0`. This happened because the exclamation mark symbol was duplicated in the generated tuned profile. With this release, the duplication no longer occurs so that inverted rules apply as intended. (link:https://issues.redhat.com/browse/OCPBUGS-35012[*OCPBUGS-35012*])
+
+* Previously, a race condition meant the Kubelet might report a false error relating to a failed resize from the volume resizer. With this release, the race condition has been fixed and the false error is no longer printed. (link:https://issues.redhat.com/browse/OCPBUGS-33964[*OCPBUGS-33964*])
+
+* Previously, escaped strings were not handled properly when editing vSphere connection, causing broken vSphere configuration. With this update, the escape strings work as expected and the vSphere configuration no longer breaks. (link:https://issues.redhat.com/browse/OCPBUGS-33942[*OCPBUGS-33942*])
+
+* Previously, when the default gateway of a node was set to `vlan` and multiple network manager connection had the same name, the node would fail as it could not configure the default OVN-Kubernetes bridge. With this release, the `configure-ovs.sh` shell script includes an `nmcli connection show uuid` command that retrieves the correct network manager connection if many connections with the same name exist. (link:https://issues.redhat.com/browse/OCPBUGS-33590[*OCPBUGS-33590*])
```

Comment:
I think connection should be plural here:
```suggestion
* Previously, when the default gateway of a node was set to `vlan` and multiple network manager connections had the same name, the node would fail as it could not configure the default OVN-Kubernetes bridge. With this release, the `configure-ovs.sh` shell script includes an `nmcli connection show uuid` command that retrieves the correct network manager connection if many connections with the same name exist. (link:https://issues.redhat.com/browse/OCPBUGS-33590[*OCPBUGS-33590*])
```